dev/src/index.js

```diff
@@ -591,15 +591,35 @@ class Firestore {
    * with an array of CollectionReferences.
    *
    * @example
+   * firestore.listCollections().then(collections => {
+   *   for (let collection of collections) {
+   *     console.log(`Found collection with id: ${collection.id}`);
+   *   }
+   * });
+   */
+  listCollections() {
+    let rootDocument = new DocumentReference(this, this._referencePath);
```

Comment:
Use `const` instead of `let` if it isn't re-assigned.  The linter should be catching this 🤔

Reply:
Done